kano_updater/ui/install_window.py

```diff
@@ -54,6 +62,29 @@ def __init__(self):
 
         self._start_install()
 
+    def _launch_game(self, window=None, event=None):
+        try:
+            if event and event.get_keycode()[1] == 44:  # [J] key
```

Comment:
I think this get_keyval should return the code that will correspond with the constant instead of the HW scan code: https://lazka.github.io/pgi-docs/index.html#Gdk-3.0/unions/Event.html#Gdk.Event.get_keyval